backgroundColor.js

```diff
@@ -0,0 +1,20 @@
+
+var interval = 1000;
+var colorUsedList = [];
+
+var changingColors = window.setInterval(changeBackgroundColor, interval);
+
+function changeBackgroundColor(){
+    var colorCode;
+    do {
+        colorCode = Math.floor(Math.random() * 1000000 + 1);
```

Comment:
`1000000` seems to be an arbitrary number, is there another number that has some significance?